src/test/resources/pages/instructorFeedbackConstSumRecipientQuestionEditSuccess.html

```diff
@@ -1272,4 +1269,12 @@ <h4 class="modal-title">
                     </div>
                 </div>
             </div>
+            <input name="questionnum" type="hidden" value="2"/>
+            <input name="fsname" type="hidden" value="First Session"/>
+            <input name="courseid" type="hidden" value="FConstSumRecipientQnUiT.CS2104"/>
+            <input name="showresponsesto" type="hidden" value="RECEIVER,INSTRUCTORS"/>
+            <input name="showgiverto" type="hidden" value="RECEIVER,INSTRUCTORS"/>
+            <input name="showrecipientto" type="hidden" value="RECEIVER,INSTRUCTORS"/>
+            <input name="user" type="hidden" value="FConstSumRecipientQnUiT.instructor"/>
+            <input id="generatedOptions" name="generatedOptions" type="hidden" value="NONE"/>
```

Comment:
The old jsp code had them outside their proper tags.

If I recall right, it's something like
````
<div>
    <form>
</div>
        <input />...<input />
    </form>
````
With the closing form tag being auto-corrected by the browser at run-time.
(the jsp file had improper indentation at first, naturally)